.cargo/audit.toml

```diff
@@ -1,7 +0,0 @@
-# https://github.com/RustSec/cargo-audit/blob/master/audit.toml.example
-
-[advisories]
-ignore = [
-    "RUSTSEC-2021-0127", # serde_cbor
-    "RUSTSEC-2021-0145", # atty
```

Comment:
These dependencies were pulled by the old Criterion version, so we no longer need these exceptions.